Library/Formula/decklinksdk.rb

```diff
@@ -0,0 +1,20 @@
+require 'formula'
+
+class Decklinksdk < Formula
+  homepage 'http://www.blackmagicdesign.com/support/sdks/'
+  url 'http://software.blackmagicdesign.com/SDK/Blackmagic_DeckLink_SDK_10.1.4.zip'
```

Comment:
Maybe it's worth skipping the example tools and just shipping the include dir?

I noticed the include dir has some .cpp files along with .h files - are those meant to be installed in "include"?